backend/lib/calculate/adequacy.py

```diff
@@ -11,8 +12,7 @@
 
 from sqlalchemy.orm import sessionmaker
 
-# TODO - Use config or environment variable.
-MEASURER = distance.get_measure('haversine')
+MEASURER = distance.get_measure(config.get('measurer'))
 EXIT_DISTANCE = 10.0    # Measured in miles.
 RELEVANCY_RADIUS_IN_METERS = 24140.2    # 15 miles in meters. Do not go lower than this.
```

Comment:
Please use miles and convert somewhere else. Also move to config maybe?

Reply:
Earlier we discussed moving the backend to meters and having the frontend do the conversions; I'm still on board with that idea.

Reply:
Ok, maybe create a ticket for that?

Reply:
Added a card.